content/docs/introduction/central.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Open mSupply Central Server"
+description = "Open mSupply central server functionality"
+date = 2022-06-10T11:38:00+00:00
+updated = 2022-06-10T11:38:00+00:00
+draft = false
+weight = 8
```

Comment:
<img width="239" alt="Screenshot 2024-04-24 at 15 28 54" src="https://github.com/msupply-foundation/msupply_docs/assets/61820074/5c664bc8-5fab-439e-8b41-ec6d8765b682">
Feels like this should be grouped by areas instead of alphabetical... Dashboard is 3rd from top and if you follow the navigation links at the bottom it doesn't make sense...